src/vs/workbench/contrib/notebook/browser/view/cellParts/cellComments.ts

```diff
@@ -155,13 +165,11 @@ export class CellComments extends CellContentPart {
 	}
 
 	override prepareLayout(): void {
-		if (this.currentElement && this._commentThreadWidget.value) {
-			this.currentElement.commentHeight = this._calculateCommentThreadHeight(this._commentThreadWidget.value.getDimensions().height);
-		}
+		this._updateHeight();
 	}
 
 	override updateInternalLayoutNow(element: ICellViewModel): void {
-		if (this.currentElement && this._commentThreadWidget.value) {
+		if (this.currentElement) {
```

Comment:
Does the length of `this._commentThreadWidgets` need to be checked here?

Reply:
Thanks for checking.

I am not exactly sure what the original intention was here.
When there is no widget, nothing would be rendered, and it does not matter where the empty container is positioned.
Maybe the goal was to avoid an assignment to style.top which may trigger a layout or something like this?

I think it would also be fine to check that this._commentThreadWidget is not empty and otherwise leave the empty container whereever it was before. Maybe that is more in line with the original logic. 
Let me know if you'd like me to do that.

Reply:
Let's leave it as is. @rebornix, if you know of a reason we need to make the check then we can add it later.